inference-engine/thirdparty/clDNN/tests/test_cases/fused_conv_eltwise_gpu_test.cpp

```diff
@@ -137,6 +140,229 @@ TEST(fused_conv_eltwise, yolov5_fused_eltw_pattern01_b_fs_yx_fsv16_f32)
+        data("prod_input2", prod_input2),
+        eltwise("eltwise1", "prod_input1", "prod_input2", eltwise_mode::prod),
+        convolution("conv", "input", { "weights" }),
+        eltwise("eltwise2", "conv", "sum_input", eltwise_mode::sum),
+        eltwise("eltwise3", { "eltwise2", "conv", "eltwise1" }, eltwise_mode::prod),
+        reorder("out_reorder", "eltwise3", format::b_fs_yx_fsv16, data_types::f32));
+
+    std::cout << "*************************************************************" << std::endl;
+    std::cout << "Test : fused_conv_eltwise_trial_new_pattern02_b_fs_yx_fsv16_fp32" << std::endl;
+    std::cout << "input : f32, b_fs_yx_fsv16, {1, 128, 40, 40}" << std::endl;
+    std::cout << "weights : f32, os_is_yx_osv16_isv16 {128, 128, 1, 1}" << std::endl;
+    std::cout << "sum_input : f32, b_fs_yx_fsv16 {1, 1, 1, 1}" << std::endl;
+    std::cout << "prob_input1 : f32, b_fs_yx_fsv16 {1, 128, 40, 40}" << std::endl;
+    std::cout << "prob_input2 : f32, b_fs_yx_fsv16 {1, 128, 40, 40}" << std::endl;
+
+    std::cout << "topology topology(" << std::endl;
+    std::cout << "    input_layout(\"input\", input.get_layout())," << std::endl;
+    std::cout << "    data(\"sum_input\", sum_input)," << std::endl;
+    std::cout << "    data(\"weights\", weights)," << std::endl;
+    std::cout << "    data(\"prod_input1\", prod_input1)," << std::endl;
+    std::cout << "    data(\"prod_input2\", prod_input2)," << std::endl;
+    std::cout << "    eltwise(\"eltwise1\", \"prod_input1\", \"prod_input2\", eltwise_mode::prod)," << std::endl;
+    std::cout << "    convolution(\"conv\", \"input\", { \"weights\" })," << std::endl;
+    std::cout << "    eltwise(\"eltwise2\", \"conv\", \"sum_input\", eltwise_mode::sum)," << std::endl;
+    std::cout << "    eltwise(\"eltwise3\", {\"eltwise2\", \"conv\", \"eltwise1\"}, eltwise_mode::prod)," << std::endl;
+    std::cout << "    reorder(\"out_reorder\", \"eltwise3\", format::bfyx, data_types::f32));" << std::endl << std::endl;
+
+    build_options opt_act;
+    // opt_act.set_option(build_option::graph_dumps_dir("/home/yblee/conv_fusing"));
+    opt_act.set_option(build_option::optimize_data(true));
+    network network_act(engine, topology_act, opt_act);
+    network_act.set_input_data("input", input);
+
+    auto outputs_act = network_act.execute();
+    EXPECT_EQ(outputs_act.size(), size_t(1));
+    EXPECT_EQ(outputs_act.begin()->first, "out_reorder");
+
+    auto output_act = outputs_act.begin()->second.get_memory();
+    auto&& out_layout_act = output_act.get_layout();
+    auto out_act_ptr = output_act.pointer<uint8_t>();
+
+    EXPECT_EQ(out_layout_act.format, format::b_fs_yx_fsv16);
+    EXPECT_EQ(out_layout_act.size.batch[0], 1);
+    EXPECT_EQ(out_layout_act.size.feature[0], 128);
+    EXPECT_EQ(out_layout_act.size.spatial[0], 40);
+    EXPECT_EQ(out_layout_act.size.spatial[1], 40);
+
+    topology topology_ref(
+        input_layout("input", input.get_layout()),
```

Comment:
we dont need to have duplicated topology. we can share them for both fused and unfused networks.

Reply:
fixed for sharing the same topology with fused and unfused networks